package.json

```diff
@@ -137,6 +137,7 @@
     "react-app-polyfill": "^1.0.4",
     "react-dom": "^16.10.0",
     "react-modal": "^3.6.1",
+    "react-svg": "^11.0.15",
```

Comment:
@anilcs240 This package is ok 250KB size and it gets used at run time. The solution we needed was something that works on build time and converts SVG to JS files without adding anything extra to bundle size.